packages/devtools_app/test/app_size/app_size_screen_test.dart

```diff
@@ -120,6 +128,34 @@ void main() {
       expect(splitter.initialFractions[0], equals(0.67));
       expect(splitter.initialFractions[1], equals(0.33));
     });
+
+    testWidgetsWithWindowSize('builds deferred content', windowSize,
```

Comment:
can we add a test to verify we can change the app unit dropdown and that the data in the treemap / table changes when we do so?

Reply:
Yes, added!